### DISCUSSION TITLE
Pagination: Contradiction de la valeur par défaut dans le Swagger (50 vs 100)

### DISCUSSION THREAD

**Discussion by alexisthethe** [before-interop/anomalieAdresse#43] [Q&A] [answered]
Il y a une contradiction dans le Swagger sur la valeur par défaut du paramètre "limit" pour la pagination.

Dans la description du Swagger il est mentionné une valeur par défaut de 50 (https://github.com/before-interop/anomalieAdresse/blob/6d27f2875d0463f4c50561c810d0804bb190e2bf/openapi.yaml#L273C51-L273C51)

Alors que dans le schéma pour le paramètre "limit" il est mentionné une valeur par défaut de 100 (https://github.com/before-interop/common/blob/06a1fe34e205dd71c687d14078ad29cc88f6f13f/common/utils/paginate.openapi.yaml#L37C21-L37C21)

Je ne me rappelles plus de la valeur par défaut qui a été discutée. Il faudra probablement corriger une des deux valeurs.

@ggrebert

**Comment by ggrebert** [comment, marked as answer]
Merci pour le retour.
Je fais la modif pour demain.
La valeur par défaut sera 50.

**Comment by alexisthethe** [comment]
voir la PR https://github.com/before-interop/common/pull/5
